README.md

```diff
@@ -1,16 +1,33 @@
 # Monitoring front end
 
-Provides a combined front end for both monitors [CPU-tracking-monitoring-service](https://github.com/ccims/CPU-tracking-monitoring-service) and the [Error-response-monitoring-service](https://github.com/ccims/Error-response-monitoring-service).
+The application provides a combined frontend for both monitors [CPU-tracking-monitoring-service](https://github.com/ccims/CPU-tracking-monitoring-service) and the [Error-response-monitoring-service](https://github.com/ccims/Error-response-monitoring-service).
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 9.1.0.
-
-## Development server
+## How to run the application
 
 Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
 
-## Code scaffolding
+## Usage
+
+Cpu-tracking-monitoring-service:
+
+The CPU-tracking-monitoring-serivce provides the possibility to add services which are to be monitored. These services as of now only comprise the database service. The services that are added need
+to be importing the [cpu-utilization-observer](https://github.com/ccims/cpu-utilization-observer) package which exposes an endpoint that the backend of the cpu-tracking-monitoring-service scrapes for
+the cpu load. 
+
+
+Error-response-monitoring-service:
+
+The Error-response-monitoring-service lets one define certain request parameters in its UI for work checking the database service. The user can adjust the type of request (GET or POST) and the expected response 
```

Comment:
ist 'work checking' ein geläufige formulierung? denn mir ist diese formulierung bisher noch nicht untergekommen. desweiteren: korrigiert mich, falls ich mich täusche, aber habt ihr bisher nicht immer den CPU-load überprüft?

Reply:
den cpu load überprüfen wir nicht mit dem error response monitor sondern mit dem cpu utilization monitor. und mit work checking ist halt sowas gemeint wie einen service in einem realitätsnahen anwendungskontext zu testen, bisschen wie unit testing.

Reply:
ups :x 

oke. dann würde ich empfehlen das 'work checking' dem entsprechend umzufomulieren. also in die richtung '[...] checking, whether [...] works [correctly | at all | oder was auch immer... ]' 
'work checking' klingt zu sehr nach 'wir überprüfen wie viel arbeit verrichtet wurde'.

Reply:
okay ich änder das mal. wahrscheinlich failen durch den commit wieder die tests haha